optional_plugins/robot/avocado_robot/__init__.py

```diff
@@ -17,8 +17,6 @@
 import re
 import subprocess
 
-from StringIO import StringIO
-
```

Comment:
(I mean either you are doing import clenaups on all files, or you do pylint cleanup on a single file, but doing just one minor fix on a single file seems odd to me)